src/com/tigersale/userInterface/inventoryManager/EditProductView.java

```diff
@@ -150,4 +182,24 @@ private void editAmount(Product product) {
             }
         }
     }
+
+    private void editBrand(Product product) {
```

Comment:
Oh I see, you make the changes then either confirm or discard them.  That makes sense.